.drone.yml

```diff
@@ -4963,6 +4963,7 @@ steps:
   image: toolhippie/docker:latest
   commands:
   - docker build -t owncloud/phoenix:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER} .
+  - docker push owncloud/phoenix:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER}
```

Comment:
It should not be published with this tag, please define a proper tag, `:latest` could be built from master branch, a git tag should be autoi tagged, e.g. v1.1.0 gets tagged as `:1`, `:1.1` and `:1.1.0`

Reply:
having proper automated releases on tag is what we need to do . yes

- pushing a tag
- build docker image and public
- build appstore tarzg and publish as github release (marketplace upload can happen manually)